examples/react_webpack/BUILD.bazel

```diff
@@ -16,6 +16,9 @@ sass(
 ts_project(
     # Experimental: Start a tsc daemon to watch for changes to make recompiles faster.
     supports_workers = True,
+    # Show an example of how to run the daemon with a custom tsc.
+    typescript_module = "@npm//ttypescript",
```

Comment:
"module" doesn't mean anything in this context. I think it should be package like https://bazelbuild.github.io/rules_nodejs/Built-ins.html#npm_package_bin

Reply:
or maybe just `typescript = "@npm//typescript`

Reply:
Fixed!